packages/lesswrong/components/form-components/FormComponentPostEditorTagging.tsx

```diff
@@ -30,41 +102,40 @@ const FormComponentPostEditorTagging = ({value, path, document, formType, update
       link={false}
     />
   } else {
-    return <div>
-      <CoreTagsChecklist existingTagIds={Object.keys(value||{})} onTagSelected={(tag, existingTagIds)=> {
-        updateCurrentValues(
-          mapValues(
-            {tagRelevance:[tag.tagId, ...existingTagIds]},
-            (arrayOfTagIds: string[]) => toDictionary(
-              arrayOfTagIds, tagId=>tagId, tagId=>1
-            )
-          )
-        )
-      }}/>
-      <TagMultiselect
-        path={path}
-        placeholder={placeholder ?? `+ Add ${taggingNamePluralCapitalSetting.get()}`}
-        
-        value={Object.keys(value||{})}
-        updateCurrentValues={(changes) => {
-          // post tagRelevance field needs to look like {string: /
-          // number}, so even though it's extra work both here and in 
-          // the callback, we need to maintain that structure.
-          updateCurrentValues(
-            mapValues(
-              changes,
-              (arrayOfTagIds: string[]) => toDictionary(
-                arrayOfTagIds, tagId=>tagId, tagId=>1
-              )
-            )
-          )
-        }}
-      />
-    </div>
+    return (
+      <div className={classes.root}>
+        {showSubforumSection && (
+          <>
+            <h2 className={classes.subforumHeader}>Subforums</h2>
+            <p className={classes.subforumExplanation}>
+              Subforums are broad topics with a dedicated community and space for general discussion. We ensure that
+              they contain relevant posts by including posts from sub-topics automatically, and through active curation.
```

Comment:
Open to suggestions on improving this blurb, although I think it covers the key points